addons/calendar/models/calendar_event.py

```diff
@@ -719,6 +719,9 @@ def write(self, values):
 
     @api.model_create_multi
     def create(self, vals_list):
+        # Prevent sending update notification when _inverse_dates is called
+        self = self.with_context(is_new=True)
```

Comment:
context keys for business related logic is not a good idea... but meh... sending an email as a side effect of `write` is also some sort of bad idea and error prone.